src/components/sections/FAQ.section.tsx

```diff
@@ -0,0 +1,38 @@
+import { Accordion, AccordionItem } from "@components";
+
+const FAQSection = () => {
+  const faqs = [
```

Comment:
I would put `faqs` outside of the component since they are static information.

Reply:
basically you want to prevent the variable from re-render?

Reply:
```
const faqs = [
  {
    question: "🤔 So, what exactly is a hackathon?",
    answers: [
      "A hackathon is made up of two sections - a social coding event where programmers, designers, and developers collaborate in teams to solve a problem. You can either create that falls under a category hack, or make whatever your heart desires as a general hack.",
      "The second section is networking events, seminars, and presentations that happen throughout the hackathon conducted by industry and academic professionals - if you’ve been putting off learning that one programming language for a while, now might be the time to learn!",
    ],
  },
  {
    question: "🕒 Cool! When and where is HawkHacks happening?",
    answers: [
      "HawkHacks will take place for 36 hours, starting on May 13th, Friday @ 6PM EDT all the way till  May 15th, Sunday @ 2PM EDT.",
      "HawkHacks is a fully virtual event, meaning everything will be hosted and managed through Hopin, Discord, and Twitch!",
    ],
  },
  {
    question: "💰 That sounds good and all, but how much does it cost?",
    answers: [
      "Nothing! Free! Nada! Due to our amazing sponsors and support systems, attending and participating at HawkHacks is absolutely free!",
    ],
  },
];

const FAQSection = () => {
```